### ISSUE TITLE
The 'hook_every_page_top' hook did not complete for the 'form_render_skip_logic' module

### ISSUE THREAD

**Issue by mamanambiya** [ctsit/form_render_skip_logic#71]

**Comment by mamanambiya** [comment]
We are running REDCap 8.4.3 and having the following error on the FRSL module:
```
The 'hook_every_page_top' hook did not complete for the 'form_render_skip_logic' module because of the following error:

Error Message: Uncaught Error: Call to undefined method Records::getRecordListSingleDag() in /data/www/htdocs/redcap/modules/form_render_skip_logic_v3.3.10/ExternalModule.php:263
Stack trace:
#0 /data/www/htdocs/redcap/modules/form_render_skip_logic_v3.3.10/ExternalModule.php(398): FormRenderSkipLogic\ExternalModule\ExternalModule->getFormsAccessMatrix(NULL, NULL)
#1 /data/www/htdocs/redcap/modules/form_render_skip_logic_v3.3.10/ExternalModule.php(56): FormRenderSkipLogic\ExternalModule\ExternalModule->loadFRSL('record_status_d...', NULL)
#2 [internal function]: FormRenderSkipLogic\ExternalModule\ExternalModule->redcap_every_page_top('200')
#3 /data/www/htdocs/redcap/redcap_v8.4.3/ExternalModules/classes/ExternalModules.php(1294): call_user_func_array(Array, Array)
#4 /data/www/htdocs/redcap/redcap_v8.4.3/ExternalModules/classes/ExternalModules.php(1377): ExternalModules\ExternalModules::startHook('form_render_ski...', 'v3.3.10', Array)
#5 /data/www/htdocs/redcap/redcap_v8.4.3/Classes/Hooks.php(37): ExternalModules\External
File: /data/www/htdocs/redcap/modules/form_render_skip_logic_v3.3.10/ExternalModule.php
Line: 263


URL: https://redcap.uct.ac.za/redcap_v8.4.3/DataEntry/record_status_dashboard.php?pid=200
Server: redcap.uct.ac.za (srvslsrcp002)
Module Name: Form Render Skip Logic (form_render_skip_logic)
Module Author(s): pbc@ufl.edu, tls@ufl.edu, marlycormar@ufl.edu, tbembersimeao@ufl.edu, kyle.chesney@ufl.edu
Hook Run Time: 0 seconds
```

**Comment by pbchase** [comment]
@mamanambiya, we referenced the wrong minimum redcap version in our module.  That feature did not appear in REDCap until 8.7.2.  You'll have to upgrade your REDCap to 8.7.2 or greater to use FRSL.  Sorry about that.

**Comment by pbchase** [comment]
@ChemiKyle, please increase our minimum redcap version to 8.7.2 in config.json.

**Comment by ChemiKyle** [comment]
Addressed by PR #73
